CHANGELOG.md

```diff
@@ -1,3 +1,149 @@
+# VERSION (UNRELEASED)
```

Comment:
There's some discussion to be had about versioning scheme. Semver doesn't make much sense for an application: something like calver (calendar versioning e.g. `helix 22.03.17`, kakoune does this for example) or an incrementing integer (e.g. `helix 7`) could be good alternatives